README.md

```diff
@@ -0,0 +1,38 @@
+fx2eeprom
+=========
+
+Read and Write the EEPROM of an FX2 chip with the help of the Cypress `vend_ax.hex` firmware.
+
+`vend_ax.hex` is property of Cypress.
+
+COMPILE
+-------
+- Install libusb-1.0-0-dev on your system, e.g. `apt install libusb-1.0-0-dev` for debian.
+- Run `make`
+
+PREPARATION
+-----------
+- Install cycfx2prog, e.g. `apt install cycfx2prog`
```

Comment:
isnt fxload more common?

Reply:
It uses the deprecated `usbfs` file system, while cycfx2prog uses VID PID.